src/App.js

```diff
@@ -1,26 +1,99 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {useEffect, useRef, useState} from 'react';
+import {ACCESS_KEY, API_URL} from './credentials';
+import axios from 'axios';
 import './App.css';
+import Pagination from "./components/pagination";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    const [images, setImages] = useState([]);
+    const [page, setPage] = useState(1);
+    const loader = useRef(null);
+    const [query, setQuery] = useState('')
+
+
+    const fetchImages = () => {
+        axios.get(`${API_URL}/photos/?client_id=${ACCESS_KEY}&per_page=10&page=${page}`)
+            .then(res => setImages([...images, ...res.data]));
```

Comment:
You don't need add new data to images. 
Every new page means new data. No need to keep old data, when you use pagination